<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="0" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-22T21:31:50 SAST" name="All Test Suite" finished-at="2024-08-22T21:32:07 SAST" duration-ms="17557">
    <groups>
    </groups>
    <test started-at="2024-08-22T21:31:50 SAST" name="First Test" finished-at="2024-08-22T21:31:58 SAST" duration-ms="8485">
      <class name="webuitests.RunnerClass">
        <test-method is-config="true" signature="beforeTest(java.lang.String)[pri:0, instance:webuitests.RunnerClass@2805c96b]" started-at="2024-08-22T21:31:50 SAST" name="beforeTest" finished-at="2024-08-22T21:31:50 SAST" duration-ms="378" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeMethod(org.testng.ITestResult, java.lang.String)[pri:0, instance:webuitests.RunnerClass@2805c96b]" started-at="2024-08-22T21:31:50 SAST" name="beforeMethod" finished-at="2024-08-22T21:31:56 SAST" duration-ms="5574" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name={null} status=CREATED method=RunnerClass.validateSiteAccessibility()[pri:0, instance:webuitests.RunnerClass@2805c96b] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method is-config="true" signature="beforeMethod(org.testng.ITestResult, java.lang.String)[pri:0, instance:webuitests.RunnerClass@2805c96b]" started-at="2024-08-22T21:31:50 SAST" name="beforeMethod" finished-at="2024-08-22T21:31:56 SAST" duration-ms="5572" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name={null} status=CREATED method=RunnerClass.validateLoginFailure(java.lang.String, java.lang.String)[pri:0, instance:webuitests.RunnerClass@2805c96b] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="validateLoginFailure(java.lang.String, java.lang.String)[pri:0, instance:webuitests.RunnerClass@2805c96b]" started-at="2024-08-22T21:31:56 SAST" name="validateLoginFailure" data-provider="loginCredentials" finished-at="2024-08-22T21:31:58 SAST" duration-ms="1790" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[tester23@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[tester23]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [https://parabank.parasoft.com/parabank/index.ht] but found [https://parabank.parasoft.com/parabank/index.htm],
did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://parabank.parasoft.com/parabank/index.ht] but found [https://parabank.parasoft.com/parabank/index.htm],
did not expect to find [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at webuitests.RunnerClass.validateLoginFailure(RunnerClass.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginFailure -->
        <test-method signature="validateSiteAccessibility()[pri:0, instance:webuitests.RunnerClass@2805c96b]" started-at="2024-08-22T21:31:56 SAST" name="validateSiteAccessibility" finished-at="2024-08-22T21:31:56 SAST" duration-ms="15" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [https://parabank.parasoft.com/parabank/index.ht] but found [https://parabank.parasoft.com/parabank/index.htm]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://parabank.parasoft.com/parabank/index.ht] but found [https://parabank.parasoft.com/parabank/index.htm]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at webuitests.RunnerClass.validateSiteAccessibility(RunnerClass.java:16)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateSiteAccessibility -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:webuitests.RunnerClass@2805c96b]" started-at="2024-08-22T21:31:56 SAST" name="afterMethod" finished-at="2024-08-22T21:31:57 SAST" duration-ms="174" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:webuitests.RunnerClass@2805c96b]" started-at="2024-08-22T21:31:58 SAST" name="afterMethod" finished-at="2024-08-22T21:31:58 SAST" duration-ms="190" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- webuitests.RunnerClass -->
    </test> <!-- First Test -->
    <test started-at="2024-08-22T21:31:59 SAST" name="Second Test" finished-at="2024-08-22T21:32:07 SAST" duration-ms="8495">
      <class name="webuitests.RunnerClass">
        <test-method is-config="true" signature="beforeTest(java.lang.String)[pri:0, instance:webuitests.RunnerClass@45b9a632]" started-at="2024-08-22T21:31:59 SAST" name="beforeTest" finished-at="2024-08-22T21:31:59 SAST" duration-ms="46" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeMethod(org.testng.ITestResult, java.lang.String)[pri:0, instance:webuitests.RunnerClass@45b9a632]" started-at="2024-08-22T21:31:59 SAST" name="beforeMethod" finished-at="2024-08-22T21:32:05 SAST" duration-ms="6493" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name={null} status=CREATED method=RunnerClass.validateSiteAccessibility()[pri:0, instance:webuitests.RunnerClass@45b9a632] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method is-config="true" signature="beforeMethod(org.testng.ITestResult, java.lang.String)[pri:0, instance:webuitests.RunnerClass@45b9a632]" started-at="2024-08-22T21:31:59 SAST" name="beforeMethod" finished-at="2024-08-22T21:32:05 SAST" duration-ms="6065" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name={null} status=CREATED method=RunnerClass.validateLoginFailure(java.lang.String, java.lang.String)[pri:0, instance:webuitests.RunnerClass@45b9a632] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="validateLoginFailure(java.lang.String, java.lang.String)[pri:0, instance:webuitests.RunnerClass@45b9a632]" started-at="2024-08-22T21:32:05 SAST" name="validateLoginFailure" data-provider="loginCredentials" finished-at="2024-08-22T21:32:06 SAST" duration-ms="907" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[tester23@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[tester23]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [https://parabank.parasoft.com/parabank/index.ht] but found [https://parabank.parasoft.com/parabank/index.htm],
did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://parabank.parasoft.com/parabank/index.ht] but found [https://parabank.parasoft.com/parabank/index.htm],
did not expect to find [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at webuitests.RunnerClass.validateLoginFailure(RunnerClass.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginFailure -->
        <test-method signature="validateSiteAccessibility()[pri:0, instance:webuitests.RunnerClass@45b9a632]" started-at="2024-08-22T21:32:05 SAST" name="validateSiteAccessibility" finished-at="2024-08-22T21:32:05 SAST" duration-ms="11" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [https://parabank.parasoft.com/parabank/index.ht] but found [https://parabank.parasoft.com/parabank/index.htm]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://parabank.parasoft.com/parabank/index.ht] but found [https://parabank.parasoft.com/parabank/index.htm]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at webuitests.RunnerClass.validateSiteAccessibility(RunnerClass.java:16)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateSiteAccessibility -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:webuitests.RunnerClass@45b9a632]" started-at="2024-08-22T21:32:06 SAST" name="afterMethod" finished-at="2024-08-22T21:32:07 SAST" duration-ms="1269" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:webuitests.RunnerClass@45b9a632]" started-at="2024-08-22T21:32:06 SAST" name="afterMethod" finished-at="2024-08-22T21:32:07 SAST" duration-ms="1302" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- webuitests.RunnerClass -->
    </test> <!-- Second Test -->
  </suite> <!-- All Test Suite -->
</testng-results>
